packages/api/src/controllers/project.ts

```diff
@@ -139,4 +139,47 @@ app.post("/", authorizer({}), async (req, res) => {
   res.json(project);
 });
 
+app.patch("/:id", authorizer({}), async (req, res) => {
+  const project = await db.project.get(req.params.id, {
+    useReplica: false,
+  });
+
+  if (!project || project.deleted) {
+    throw new NotFoundError(`project not found`);
+  }
+
+  if (req.user.admin !== true && req.user.id !== project.userId) {
+    throw new ForbiddenError("user can only update their own projects");
+  }
+
+  const { name } = req.body;
+
+  await db.project.update(req.params.id, {
+    name: name,
+  });
+
+  res.status(204);
+  res.end();
+});
+
+app.delete("/:id", authorizer({}), async (req, res) => {
+  const project = await db.project.get(req.params.id, {
+    useReplica: false,
+  });
+
+  if (!project || project.deleted) {
+    throw new NotFoundError(`project not found`);
+  }
+
+  if (req.user.admin !== true && req.user.id !== project.userId) {
+    throw new ForbiddenError("user can only delete their own projects");
+  }
+
+  //await db.project.delete(req.params.id);
```

Comment:
1. Could you please remove the commented-out code? 
2. How do we plan to address removing projects? Do we want to delete all related resources, but it takes too long so we don't do it here? What's the plan around it?

Reply:
1. Sure!
2. The plan is to wait for this https://github.com/livepeer/studio/pull/2174 and make the deletion a cron job that lists files in queue to get deleted and do it in batches. We don't want to just delete them here, because they could be thousands